helper/ldaputil/client.go

```diff
@@ -428,7 +428,7 @@ func getCN(dn string) string {
 
 	for _, rdn := range parsedDN.RDNs {
 		for _, rdnAttr := range rdn.Attributes {
-			if rdnAttr.Type == "CN" {
+			if strings.ToLower(rdnAttr.Type) == "cn" {
```

Comment:
`strings.EqualFold()` would work well here.

Reply:
Thanks, changed to `strings.EqualFold()`